instance/tests/integration/integration_instance.py

```diff
@@ -107,6 +110,34 @@ def test_external_databases(self):
         self.assertTrue(instance.successfully_provisioned)
         self.assertFalse(instance.require_user_creation_success())
 
+    def test_activity_csv(self):
+        """
+        Run the activity_csv management command against a live instance.
+        """
+        OpenEdXInstanceFactory(name='Integration - test_spawn_appserver')
+        instance = OpenEdXInstance.objects.get()
+        spawn_appserver(instance.ref.pk, mark_active_on_success=True, num_attempts=2)
+        self.assert_instance_up(instance)
+        self.assertTrue(instance.successfully_provisioned)
+        self.assertTrue(instance.require_user_creation_success())
+
+        out = StringIO()
+        with patch('sys.exit') as mock_exit:
+            call_command('activity_csv', stdout=out)
+            mock_exit.assert_called_with(0)
+
+        out_lines = out.getvalue().split('\n')
+
+        # The output should look similar to this when one instance is launched:
+        #
+        #   Appserver IP,Owner Emails,Unique Hits,Total Users,Total Courses,Age (Days)
+        #   149.202.188.116,"['brandon@opencraft.com', 'x@bdero.me']",4,7,1,0
+
+        csv.reader(out_lines)
```

Comment:
@bdero Do we need this line? It looks like `csv.reader` isn't called again after this (the assertions check `out_lines` directly).